script/legacy/2019-05-14-armagaddon/delete-from-bq.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+## Affected period to delete is
+## 2019-05-04T11:00:00Z to 2019-05-11T11:00:00Z
```

Comment:
I misread this time range and thought it was only a single day. I'm working on updating to be the entire week as stated.